### PR TITLE
Add lane duplicate calculation to the run stage

### PR DESCRIPTION
**Pull request by tcezard**

### CODE REVIEW

File: analysis_driver/constants.py [hunk 1]
```diff
@@ -24,6 +24,7 @@
 ELEMENT_NB_BASE_R2_CLEANED = 'clean_bases_r2'
 ELEMENT_NB_Q30_R1_CLEANED = 'clean_q30_bases_r1'
```
Comment (katieemelianova) [inline analysis_driver/constants.py:25]: Needs the latest reporting app pull request to be put into production before duplicate calculation can be used

**Comment by tcezard** [comment]
There is a problem in testing this PR because well duplicate search for all the tiles and will crash if one is missing.
We need to add -t 1101 to the command line to make it work with test runs.

**Comment by tcezard** [comment]
This PR was tested on Ultra and works.
Can this be reviewed?

**Comment by katieemelianova** [comment]
Hi Tim,
Sorry it slipped my mind yesterday, I'll do that today!
Best,
Katie

Date: Mon, 9 May 2016 22:12:11 -0700

**Pull request merged**